caldp/create_previews.py

```diff
@@ -481,6 +486,12 @@ def create_previews(self, input_paths):
 PREVIEW_MANAGERS = {"ipst": IpstPreviewManager, "svm": HapPreviewManager, "mvm": HapPreviewManager}
 
 
+def find_file(name, path):
```

Comment:
There is a very small but non-zero chance that this function becomes a performance problem in the future. I'm worried that it walks the path every time the function is called, and if the path is large, that ends up being a big waste of time if it's called many times on the same path. It could also just become a problem if the path it's walking is huge for some reason or another.

Reply:
It's currently only used to find the ACS_WFC3_filters.txt file, in case we move it around. It's only called once when creating previews so shouldn't be a problem now. I agree that if we start using this function more in the future we'll have to be careful about how much time it takes. I'll make a note of it.